### ISSUE TITLE
more documentation please

### ISSUE THREAD

**Issue by lincolnharmer** [rohanpai/Robinhood#3]
I'm having trouble with intsallation

**Comment by whereisjim** [comment]
Can you please update on how to do the installation?

**Comment by mikeseese** [comment]
I agree. You reference a "requirements.txt" file for pip, and that file does not exist in the repo. I've decided to give up on this API haha.

**Comment by benkroop** [comment]
While requirements.txt doesn't exist, with a little digging I was able to easily log in, get quotes, and place a buy order using Robinhood.py. As the code is currently you won't be able to place a sell order. 

The only package you should need to install is requests. If you have trouble getting the code to work, make sure you've changed the account number in the URL on line 79 of Robinhood.py to your account number, minus the last two digits. This can be found on your PDF statement - accessible through the app.

Directions:
1. pip install requests
2. Update your account number in the URL on line 79.
3. Update the example.py to use your username and password. I recommend updating the example so that your username and password aren't hardcoded into the .py file, but you can leave it hardcoded if you want.
4. Comment out all commands you don't want to run. The example automatically buys and then sells a share of the stock you select, and you may not want that. You can also add print statements to view stock quote data.
5. example.py should run successfully and place orders.
